adbms/src/main/scala/de/up/hpi/informationsystems/adbms/definition/Exceptions.scala

```diff
@@ -31,3 +31,10 @@ case class RecordNotFoundException(message: String) extends AdbmsException(messa
   * @param message gives details
   */
 case class InconsistentStateException(message: String) extends AdbmsException(message)
+
+/**
+  * Indicates a failed authentication attempt with an authentication authority.
+  *
+  * @param message gives details
+  */
+case class AuthenticationFailedException(message: String) extends AdbmsException(message)
```

Comment:
This is an application defined exception, right?
Currently it's not used in the framework, so I would put this definition in the `sampleapp`